openstack_controller/README.md

```diff
@@ -13,6 +13,14 @@ need to install anything else on your server.
 
 ### Configuration
 
+#### Prepare OpenStack
+
+Create a datadog user that will be specified in your `openstack_controller.d/conf.yaml` file. This user requires admin read only permissions across your environment so that it can be run from a single node and read high level system information about all compute nodes and servers. 
```

Comment:
```suggestion
Create a `datadog` user this is used in your `openstack_controller.d/conf.yaml` file. This user requires admin read only permissions across your environment so that it can be run from a single node and read high level system information about all compute nodes and servers. 
```

Reply:
^ Remove future tense from docs.

Reply:
Done, thanks 🙇